desktop/src/main/resources/images.css

```diff
@@ -100,3 +100,11 @@
 #governance {
     -fx-image: url("images/navigation/governance.png");
 }
+
+#welcome-community {
+    -fx-image: url("images/welcome-community@2x.png");
```

Comment:
The @2x postfix should not be added to let the javafx framework apply the right image based on the resolution.
We just need to have both images provided in the resources. One with `@2x` postfix for high resolution and one without.

Reply:
Fixed